.github/workflows/build-and-test-tool.yml

```diff
@@ -39,60 +39,60 @@ jobs:
       MORYX_PACKAGE_TARGET_V3_RELEASE: ${{ env.MORYX_PACKAGE_TARGET_V3_RELEASE }}
 
   Build:
-    needs: [EnvVar]
-    uses: phoenixcontact/tools/.github/workflows/build-tool.yml@main
+    needs: [SetupEnvironment]
+    uses: ./.github/workflows/build-tool.yml
```

Comment:
Better to keep the reference to the tools-repo, we don't need this specific workflow in this repo